Packs/AzureNetworkSecurityGroups/Integrations/AzureNetworkSecurityGroups/AzureNetworkSecurityGroups.py

```diff
@@ -497,6 +573,278 @@ def nsg_resource_group_list_command(client: AzureNSGClient, params: Dict, args:
     )
 
 
+@logger
+def azure_nsg_public_ip_addresses_list(client: AzureNSGClient, params: Dict, args: Dict) -> CommandResults:
```

Comment:
```suggestion
def azure_nsg_public_ip_addresses_list_command(client: AzureNSGClient, params: Dict, args: Dict) -> CommandResults:
```
Rename all command functions to end with `_command`